src/Chapter06.Tests/Listing06.36.UsingNotNullWhenAndNotNullIfNotNullAttributes.Tests.cs

```diff
@@ -1,34 +1,33 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 
-namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_36.Tests
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter06.Listing06_36.Tests;
+
+[TestClass]
+public class NullabilityAttributesExaminedTests
 {
-    [TestClass]
-    public class NullabilityAttributesExaminedTests
+    [TestMethod]
+    public void TryGetAsText_Given4_ReturnFour()
     {
-        [TestMethod]
-        public void TryGetAsText_Given4_ReturnFour()
+        if(NullabilityAttributesExamined.TryGetDigitAsText('4', out string? text))
         {
-            if(NullabilityAttributesExamined.TryGetDigitAsText('4', out string? text))
-            {
-                Assert.AreEqual<string>("four", text);
-            }
+            Assert.AreEqual<string>("four", text);
         }
+    }
```

Comment:
You shouldn't have conditions in the unit test. Better to Assert that `TryGetDigitAsText` returns true, and then assert that `text` equals `"four"`